### ISSUE TITLE
Enabling read/write on issue

### ISSUE THREAD

**Issue by grace110693** [bitnami/charts#9775]
### Name and Version

1.12.0

### What steps will reproduce the bug?

1. MariaDB running in kubernetes pod

### Are you using any custom parameters or values?

 datasource:
        url: jdbc:mysql:sequential://maria-mariadb-primary:3306,maria-mariadb-secondary:3306/grcont?failOverReadOnly=true

### What is the expected behavior?

When master fails, slave is opened on read/write mode

### What do you see instead?

slave is expected to be in read only

**Comment by javsalgar** [comment]
Hi,

I see you created a similar issue. Could you let us know some detailed steps to reproduce the issue? The values you changed from the default, some steps to force the situation you desrcribe. That would be helpful to us to identify the issue.

**Comment by grace110693** [comment]
1. We deployed the bitnami chart with default configs
2. MariaDb is deployed with primary & secondary with service name "maria-mariadb-primary"& "maria-mariadb-secondary"
3. We used these service name in java boot spring application.properties "datasource:
url: jdbc:mysql:sequential://maria-mariadb-primary:3306,maria-mariadb-secondary:3306/grcont?failOverReadOnly=true"
4. After this we expected when primary fails, secondary will be opened in read mode, but it still open in write mode
5. And also till before primary fails, primary and secondary are in sync, but after the failover and primary is recovered, they are never in sync, unless redeployed

Our expectations is that by default MariaDB master is opened in read/write mode and slave in read this doesn't seem to be the case and also when we set config explicitly also (as mentioned in point 3) its not being changed. Essentially we don't seem to have control of the mode of the secondary after primary fails.

**Comment by miguelaeh** [comment]
Hi @grace110693 ,

> After this we expected when primary fails, secondary will be opened in read mode, but it still open in write mode

Secondary should not allow writes. It has a separated statefulset which in principle should not be promoted to primary. Could you share some logs of that promotion you mention please?
